contrib/python/conan_helpers.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import os
+
+def parse_chromium_readme(file_path):
```

Comment:
Nit: This and `patch_chromium_readme` could be prefixed with _ as they are internal helper methods and do not need to be public

Reply:
Good point. Done. Thanks!